tests/ui/ok_if_let.rs

```diff
@@ -16,8 +18,18 @@ fn str_to_int_ok(x: &str) -> i32 {
     }
 }
 
+fn nested_some_no_else(x: &str) -> i32 {
+    {
+        if let Some(y) = x.parse().ok() {
+            return y;
+        };
+        0
+    }
+}
+
 fn main() {
-    let y = str_to_int("1");
-    let z = str_to_int_ok("2");
-    println!("{}{}", y, z);
+    let x = str_to_int("1");
+    let y = str_to_int_ok("2");
+    let z = nested_some_no_else("3");
+    println!("{}{}{}", x, y, z);
```

Comment:
`let _` and remove the println!

Reply:
removed via d8ac4b0195e7c4cb6667a9cc67c14c3df06b7322